minigalaxy/ui/properties.py

```diff
@@ -56,6 +60,7 @@ def ok_pressed(self, button):
             self.game.set_info("variable", str(self.entry_properties_variable.get_text()))
             self.game.set_info("command", str(self.entry_properties_command.get_text()))
             self.game.set_info("show_fps", self.switch_properties_show_fps.get_active())
+            self.game.set_info("hide_game", self.switch_properties_hide_game.get_active())
```

Comment:
This doesn't need to be under "if" condition (should work for not installed games too).

Once implementation is done in __filter_library_func() use filter_library() here.